src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import duke.tasks.TaskList;
+
+import java.io.*;
+
+/**
+ * Represents the file containing the list of tasks.
+ */
+public class Storage {
+    private static final String text = "./data/duke.txt";
```

Comment:
Since the `text` variable represents a constant file path, perhaps you can rename it to `FILE_PATH` to adhere to Java coding standard and makes it more descriptive.